resources/conf/logback/logback.properties

```diff
@@ -1,17 +0,0 @@
-#
```

Comment:
Same point as above, is the reason we are deleting this whole file because it's already pre-configured in Nexus 3?

Reply:
Yes, same with the above point